src/country.ts

```diff
@@ -48,6 +48,10 @@ class Country {
     return this.data.country_code ?? this.data.dial_code
   }
 
+  public get partOf(): string[] | null {
```

Comment:
Shall we return empty array when not defined?

Reply:
Fixed